packages/client-core/src/admin/components/Location/LocationDrawer.tsx

```diff
@@ -91,12 +92,24 @@ const LocationDrawer = ({ open, mode, selectedLocation, onClose }: Props) => {
   const hasWriteAccess = user.scopes.get(NO_PROXY)?.find((item) => item?.type === 'location:write')
   const viewMode = mode === LocationDrawerMode.ViewEdit && !editMode.value
 
-  const sceneMenu: InputMenuItem[] = scenes.get(NO_PROXY).map((el) => {
-    return {
-      value: `${el.project}/${el.name}`,
-      label: `${el.name} (${el.project})`
-    }
-  })
+  const sceneName = selectedScene ? selectedScene.split('/')[1] : ''
+  const projectName = selectedScene ? selectedScene.split('/', 1)[0] : ''
+
+  if (selectedScene) state.scene.set(selectedScene)
```

Comment:
we shouldn't be setting state in the body of a react component. it should be in a useEffect listening for changes to selectedScene